packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -242,8 +233,6 @@ export class DotnetPluginImpl implements PluginImpl {
     let clientSecret = Placeholders.clientSecret;
     let tenantId = "";
     let oauthAuthority = Placeholders.oauthAuthority;
```

Comment:
There is a Code scanning warning here, please fix it as GitHub suggested.
![image](https://user-images.githubusercontent.com/2046804/168800278-efd74841-e540-40a7-97d2-5d8bfe65e3cd.png)

Reply:
Got it